src/esp/gfx/TextRenderer.h

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <Corrade/Containers/Containers.h>
+#include <Corrade/PluginManager/Manager.h>
+#include <Magnum/Math/Color.h>
+#include <Magnum/Math/Matrix3.h>
+#include <Magnum/Shaders/Vector.h>
+#include <Magnum/Text/AbstractFont.h>
+#include <Magnum/Text/Alignment.h>
+#include <Magnum/Text/GlyphCache.h>
+#include <Magnum/Text/Renderer.h>
+#include <string>
+#include <vector>
+
+namespace esp {
+namespace gfx {
+
+namespace Cr = Corrade;
+namespace Mn = Magnum;
+
+const std::string defaultCharacters =
+    "abcdefghijklmnopqrstuvwxyz"
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    "0123456789?!:;,.# ";
+
+class TextRenderer {
+ public:
+  /**
+   * @brief Constructor
+   */
+  // aspectRatio = width / height
+  explicit TextRenderer(float viewportAspectRatio = 1.33333f,
+                        float fontSize = 20.0f,
+                        int cacheSize = 512,
+                        const std::string& characters = defaultCharacters);
```

Comment:
Sorry, my bad. Yes, it is amateur. It will become a disaster when the header is included by multiple .cpp.
Good catch. I wish I can came up with the same idea as yours to prevent such silly mistake.